### DISCUSSION TITLE
Is aarch64 supported?

### DISCUSSION THREAD

**Discussion by dreams-money** [coinfoundry/miningcore#1517] [General]
Hello,

Is aarch64 supported at all?  Reason I ask - I attempted to build the project on an ARM machine and the Dotnet requirements seem to have halted that.

**Comment by oliverw** [comment]
It should work for some coins but not for all. The reason is not .NET compatibility but lots of the native code MC uses contains x86 specific optimizations. Monero for example.

**Reply by oliverw** [reply]
Actually I meant `libcryptonight` and `libcryptonote`.

**Reply by oliverw** [reply]
Most likely also `librandomx`.

**Comment by dreams-money** [comment]
Thank you so much for the response. My mistake on the .NET dependencies as well. I did compile past that last night.

Thank you for the Monero detail as well - that was my latest ARM roadblock. I'll try it without Monero.
